src/components/Card.tsx

```diff
@@ -11,32 +13,15 @@ const Card = ({ cardData }: CardProps) => {
   const [pathname, setPathName] = useState(window.location.pathname)
 
   return (
-    <div className="m-8 text-center">
-      <figure className="relative rounded-xl figure-effect">
-        <img
-          src={cardData.images.small}
-          className="figure-img rounded-xl custom-box-shadow  w-64 h-80 transition-all"
-        />
-        <figcaption
-          className="figure-text 
-        text-white 
-        bg-gray-800 
-        p-8
-        absolute 
-        object-left 
-        top-0 h-full 
-        w-72 
-        opacity-0 
-        rounded-lg 
-        custom-box-shadow 
-        transition-all 
-        duration-300"
-        >
-          <span className="figure-inner-text">
+    <div tw="m-8 text-center">
+      <Figure css={[figEffectHover]}>
```

Comment:
instead of passing this via css prop, just add it to your styled-component directly